src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,22 @@
 const generalCSQuiz = [
+  {
+    Question: "What is the acceptable range of Balance Factor for an AVL tree?",
+    Answer: "[-1,1]",
+    Distractor1: "[0,1]",
+    Distractor2: "[-1,0]",
+    Distractor3: "None of the above",
+    Explanation: " The self balancing property of an AVL tree is maintained by the balance factor. The value of balance factor should always be -1, 0 or +1",
+    Link: "https://forum.freecodecamp.org/t/algorithms-avl-trees/297458"
+  },
+  {
+    Question: "What method copies the actual value of an argument into the formal parameter of the function?",
```

Comment:
This would be better:
```suggestion
    Question: "Which approach involves duplicating the argument's value into the function's formal parameter?",
```